packages/rest-api-client/src/client/types/space/index.ts

```diff
@@ -43,6 +43,20 @@ export type Space = {
   permissions: Permissions;
 };
 
+export type UpdateSpaceForRequest = {
+  id: SpaceID;
+  name: string;
+  isPrivate: boolean;
+  useMultiThread: boolean;
+  fixedMember: boolean;
+  showAnnouncement: boolean;
+  showThreadList: boolean;
+  showAppList: boolean;
+  showMemberList: boolean;
+  showRelatedLinkList: boolean;
+  permissions: Permissions;
```

Comment:
スペースID以外は省略可能なのでOptionalですね！

```suggestion
  name?: string;
  isPrivate?: boolean;
  useMultiThread?: boolean;
  fixedMember?: boolean;
  showAnnouncement?: boolean;
  showThreadList?: boolean;
  showAppList?: boolean;
  showMemberList?: boolean;
  showRelatedLinkList?: boolean;
  permissions?: Permissions;
```

https://cybozu.dev/ja/kintone/docs/rest-api/spaces/update-space/#request

Reply:
ありがとうございます、見逃してました
[89e6ee6](https://github.com/kintone/js-sdk/pull/2884/commits/89e6ee60c50ae58831bfe26484917a7cb28ecc69)